front/src/modules/rollingStock/components/RollingStockSelector/WithRollingStockSelector.tsx

```diff
@@ -11,33 +11,33 @@ const WithRollingStockSelector =
   (hocProps: T) => {
     const { t } = useTranslation(['translation', 'rollingstock']);
 
-    const [rollingStockSelected, setRollingStockSelected] = useState<RollingStock | undefined>(
-      undefined
-    );
-    const rollingStockID = useSelector(getRollingStockID);
+    const [rollingStockSelected, setRollingStockSelected] = useState<
+      RollingStockWithLiveries | undefined
+    >(undefined);
```

Comment:
Only a small suggestion :) 
```suggestion
    >();
```